### PR TITLE
Update SP5: Auth comma and validation text

### PR DESCRIPTION
**Pull request by tiffanynolan**
- Added more info to make it more clear and obvious that users need to add a comma prior to the new auth section they add. 

- Replaced the verification text with the attribute instead.

### CODE REVIEW

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 1]
```diff
@@ -207,7 +207,18 @@ Follow these steps to create a client for the front-end app.
...
 +
-NOTE: The auth section should be at the same level as the existing backend section and indented to match. You will also need to add a comma after the prior section's }, so it should look like `},`.
+IMPORTANT: *On the line above where you add the new `"auth":` section, make sure you add a comma (`,`) after that section's closing `}`.*
++
+It should look like:
++
+[subs="attributes+"]
+----
+},
+"auth:" {
```
Comment (pb82) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: this is not rendered as two lines in the solution explorer, but its still valid json

Comment (pb82) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: @tiffanynolan I'd rather remove this section and rely on the example in the `Check your work` box. What do you think?

Comment (tiffanynolan) [reply walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: Agreed, if it's not rendering properly. I just wanted to make sure people didn't miss that darn comma.

File: walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc [hunk 2]
```diff
@@ -207,7 +207,10 @@ Follow these steps to create a client for the front-end app.
 .. Select the *YAML* tab.
 .. Add a new auth section to the showcaseConfig. You can do this by adding `"auth":` and then pasting the content that was copied in the previous step.
 +
-NOTE: The auth section should be at the same level as the existing backend section and indented to match. You will also need to add a comma after the prior section's }, so it should look like `},`.
+IMPORTANT: *On the line above where you add the new `"auth":` section, make sure you add a comma (`,`) after that section's closing `}`.*
+
```
Comment (pb82) [inline walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc]: this newline needs to be removed

**Review by pb82 — APPROVED** [review]

**Pull request merged**